arviz/plots/autocorrplot.py

```diff
@@ -51,20 +51,20 @@ def plot_autocorr(
         automatically inferred.
     figsize: tuple
         Figure size. If None it will be defined automatically.
-        Note this is not used if ax is supplied.
+        Note this is not used if ``ax`` is supplied.
     textsize: float
         Text size scaling factor for labels, titles and lines. If None it will be autoscaled based
-        on figsize.
+        on ``figsize``.
     labeller : labeller instance, optional
-        Class providing the method `make_label_vert` to generate the labels in the plot titles.
+        Class providing the method ``make_label_vert`` to generate the labels in the plot titles.
         Read the :ref:`label_guide` for more details and usage examples.
     ax: numpy array-like of matplotlib axes or bokeh figures, optional
         A 2D array of locations into which to plot the densities. If not supplied, Arviz will create
         its own array of plot areas (and return it).
     backend: str, optional
         Select plotting backend {"matplotlib","bokeh"}. Default "matplotlib".
     backend_config: dict, optional
-        Currently specifies the bounds to use for bokeh axes. Defaults to value set in rcParams.
+        Currently specifies the bounds to use for bokeh axes. Defaults to value set in :class:`arviz.RcParams`.
```

Comment:
This should explain (unless unambiguously clear) which rcParams key i the one that defines this, and (once created if created, link to the rcParams guide). `RcParams` is not present in the api documentation (nor it should be)